packages/vendure-plugin-frequently-bought-together/src/frequently-bought-together.plugin.ts

```
@@ -35,33 +61,83 @@ import { PluginInitOptions } from './types';
+          }. Some functionality might be disabled`,
+          loggerCtx
+        );
+        FrequentlyBoughtTogetherPlugin.options.hasValidLicense = false;
+      });
+  }
+
   static init(
-    options: Partial<PluginInitOptions>
+    options: FrequentlyBoughtTogetherPluginOptions
   ): Type<FrequentlyBoughtTogetherPlugin> {
     this.options = {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)